### PR TITLE
Allow error suppression in image extracting for deep learning 

### PR DESCRIPTION
**Pull request by alexzwanenburg**

### CODE REVIEW

File: mirp/mainFunctions.py [hunk 1]
```diff
@@ -122,7 +122,16 @@ def extract_images_for_deep_learning(data_config, settings_config, output_slices
...
     image_list = []
     for data_obj in data_obj_list:
-        image_list += [data_obj.process_deep_learning(output_slices=output_slices)]
+
+        if skip_errors:
+            try:
+                image_list += [data_obj.process_deep_learning(output_slices=output_slices)]
+            except Exception as e:
+                logging.error(f"Failed to extract data for {data_obj.subject}: {e}")
+                pass
```
Comment (codingS3b) [inline mirp/mainFunctions.py:131]: not sure if 'pass' is correct here, I would have expected 'continue', but maybe they have the same effect?

Comment (alexzwanenburg) [reply mirp/mainFunctions.py:131]: Thank you for mentioning this. In fact, `pass` may not even be necessary, as we already print information using `logging.error`. `continue` should not be necessary as the loop concludes after this anyway.

Comment (codingS3b) [reply mirp/mainFunctions.py]: You are right :)

**Comment by codingS3b** [comment]
Maybe it would also be good in case of failure not only to print the subject id but also the data string since we might want to extract different modalities from the same subject and only some of them might fail.

**Pull request closed without merging**